src/SmartComponents/AdvisoryDetail/__snapshots__/AdvisoryDetail.test.js.snap

```diff
@@ -81,7 +81,7 @@ exports[`AdvisoryDetail.js Should match the snapshots 1`] = `
                 Object {
                   "isActive": false,
                   "title": "Patch advisories",
-                  "to": "/advisories",
+                  "to": undefined,
```

Comment:
Maybe the test-data/fixtures needs to be adjusted here?

Reply:
This still needs to be a 'to', but without changing the paths.path to a hardcoded string, I wasnt able to find a way where this wasnt angry about this field. It seems like it never 'gets passed down' everything/ it cant get the Route info the same as before. 

To try to elaborate, with the new set up, routes sort of render dynamically, which is why we're able to do something like paths.path in the actual js files and it understands which path to look at. I dont think the test work the same way(I could be wrong here but this seems to be to case). The test just grab the entire path array and looks for paths.path which of course fails.